app/lib/pre_assembly/digital_object.rb

```diff
@@ -172,16 +167,13 @@ def build_from_staging_location(objects:, processing_configuration:, reading_ord
                                                    reading_order:)
     end
 
-    # The reading order for books is determined by the content structure set, defaulting to 'ltr'
+    # The reading order for books is determined by what the user set when registering the object.
     # This is passed to the content metadata creator, which uses it if the content structure is book
+    # Assume left-to-right if missing in the cocina structural (which really shouldn't happen for this content type)
     def reading_order
       return unless content_md_creation_style == :simple_book
 
-      if content_structure == 'simple_book_rtl'
-        'right-to-left'
-      else
-        'left-to-right'
-      end
+      existing_cocina_object.structural&.hasMemberOrders&.first&.viewingDirection || 'left-to-right'
     end
```

Comment:
this is the core change -- get it from cocina, with a default of `left-to-right` if missing